mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -531,7 +532,7 @@ def test_read_vmrk_annotations():
 def test_read_vhdr_annotations_and_events():
     """Test load brainvision annotations and parse them to events."""
     sfreq = 1000.0
-    expected_orig_time = 1384359243.794231
+    expected_orig_time = _stamp_to_dt((1384359243, 794232))
```

Comment:
why do you have all these changes by 1uS ?

Reply:
I think in principle it should be more accurate now, as it uses `round` rather than `int` cast. These values I'm assuming were taken from what was produced by the old code, not necessarily what was most correct.